plugin.py

```diff
@@ -27,13 +29,46 @@ def process(self, trace, data_context):
         log.info(f"processing trace {trace.get('name')}")
         # Add your plugin implementation here
 
-    # @Transformer enables the HANSKEN to execute code in the plugin
-    # The transformer can be found in when you expand the plugin in the extraction tools
-    # the transformers can be accessed with the /tools/transformers URL in HANSKEN
     @transformer
-    def example_transformer(self, number: int) -> int:
+    def bytes_test(self, data: bytes) -> bytes:
```

Comment:
the point of the template is to be compact, and give pointers to all available options.
on the main branch, there is just one method with a clear comment. maybe extend the documentation there (add a single line) that other types can be used. 

Maybe you can update the type in the example to str, as that would be the most common input?

```

    @transformer
    def example_transformer(self, number: int) -> int:
        # Optional transformer method to run this method with a REST call
        # See detail at:
        #  https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/dev/python/transformers.html
        return number + 1
```